hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/DynamicConfigTemplate.java

```diff
@@ -373,10 +373,13 @@ void addMapConfig(String name, int backupCount, int asyncBackupCount, int timeTo
      *                              {@code DISCARD_NEWEST}, {@code BLOCK} and {@code ERROR}.
      * @param executor              a serialized {@link java.util.concurrent.Executor} instance to use for executing
      *                              message listeners or {@code null}
+     * @param executorClassName     the class name of a {@link java.util.concurrent.Executor} implementation to use
+     *                              for executing message listeners or {@code null}
      */
     @Request(id = 15, retryable = false, response = ResponseMessageConst.VOID)
     void addReliableTopicConfig(String name, @Nullable List<ListenerConfigHolder> listenerConfigs, int readBatchSize,
-                                boolean statisticsEnabled, String topicOverloadPolicy, @Nullable Data executor);
+                                boolean statisticsEnabled, String topicOverloadPolicy, @Nullable Data executor,
```

Comment:
can both the executor and executorClassName be provided simultaneously?

Reply:
Yes, they can. The executor instance can still be provided, nothing changed there. In addition, executorClassName can be specified, too. If both are specified, then executor takes precedence over executorClassName.